modules/pv.py

```diff
@@ -33,36 +37,50 @@
     disks:
         required: true
         description: Disks from which the Physical Volumes are to be created,
-                     or Physical Volumes that are to be removed needs to be
+                     removed, resized and changed needs to be
                      specified here.
-    options:
-        required: false
-        description: Extra options that needs to be passed while creating the
-                     Physical Volumes can be given here. Check the man page of
-                     pvcreate for more info.
     size:
-        required: true if action is resize and opertion is shrink
+        required: true
         description: Specifies to what size the physical volume is to be
                      shrunken
+    force:
+        required: false
+        description: Force  the  creation  without  any confirmation.
+    uuid:
+        required: false
+        description: Specify the uuid for the device.  Without this option,
+                     pvcreate generates a random uuid.
+    zero:
+        required: false
+        description: Whether or not the first 4 sectors of the device should
+                     be wiped.
+    metadatasize:
+        required: false
+        description: The approximate amount of space to be set aside for
+                     each metadata area.
+    dataalignment:
+        required: true
+        description: Align the start of the data to a multiple of this number.
 
-authors: Anusha Rao, Nandaja Varma
 '''
 
 EXAMPLES = '''
-#Create Physical Volumes /dev/sdb and /dev/sdc with
-#dataalignment 1280k
-    - pv: action=create disks="{{ item }}"
-          options="--dataalignment 1280k"
+#Create Physical Volumes /dev/sdb with dataalignment 1280k
+    - pv: action=create disks='["/dev/sdb"]'
```

Comment:
Here we have deprecated disks. Say disk="{{ item }}" or disk=/dev/sdb
We should not be using ["/dev/sdb"] that is wrong documentation.